src/css/custom.css

```
@@ -349,3 +349,14 @@ span.callout + p::after {
 .dmn-decision-table-container .tjs-table {
   width: 100%;
 }
+
+.theme-doc-markdown img {
+  border-style: solid;
+  border-color: lightgray;
+  border-width: 1px;
+  box-shadow: 4px 4px 4px 4px lightgray;
+}
+
+.theme-doc-markdown table img {
+  border: none;
+}
```

```suggestion
.theme-doc-markdown table img {
  border: none;
  box-shadow: none;
}
```

@christinaausley I think this is what you're looking for to remove the borders inside of tables. In addition to resetting the `border` property, we also need to reset the `box-shadow` property. 

(If you look closely at the images inside of the table, or inspect in your browser tools, you'll see that they *don't* actually have a border property; you reset it properly in this code. They *do* still have a box-shadow property, and that's what you're seeing.)

---

THANK YOU! 🥳